src/splitwise.js

```diff
@@ -600,7 +607,13 @@ const getEndpointMethodGenerator = (logger, accessTokenPromise, defaultIDs, oaut
  */
 class Splitwise {
   constructor(options = {}) {
-    const { consumerKey, consumerSecret, accessToken } = options;
+
+    const { consumerKey, consumerSecret, accessToken, useOauth2 = false, redirect_uri = null } = options;
```

Comment:
I don't think `useOauth2` is an accurate option name. whether we use client credentials or authorization code, it's oauth 2 either way. maybe we can have something like a `grantType` parameter which defaults to `client_credentials`

also let's use consistent casing with the `redirect_uri` argument